### DISCUSSION TITLE
AVALANCHE FOUNDATION stake to AVALANCHE validators

### DISCUSSION THREAD

**Discussion by mahofmahof** [avalanche-foundation/ACPs#60] [Brainstorming]
I would have been really glad if AVALANCHE FOUNDATION takes care of AVALANCHE VALIDATORS. Foundation can stake their AVAX with a fee min fee of %10 to AVALANCHE validators who puts their precious effort to validate AVALANCHE and subnets.

**Comment by minivipers** [comment]
I have been avalanche ecosystem since the beginning of denali testnet like mahof and I am running validator node by myself and my avax stake is larger than I while I started to validate avalanche mainnet. I aggree with mahof and 
 I think validators can be in 2 different categories that 
1) low stake validators may  that are (between 2000 - 10000 avax ) %10 fee and foundation may stake these validadators
2) high stake validators (above 10k avax validators) may use %2 fee and anybody who has 25 or above avax can stake their these validator nodes.

**Reply by FrancoisPaul69** [reply]
you mean the reward for the foundation is 5 times greater than the one for an average validator ? (2 to 10%)

**Comment by Blckchn4ever** [comment]
I agree with mahof. It’s obvious that any validation run under 10000 avax is only for the sake of the avalanche ecosystem. And those taking the burden should be rewarded for standing for all during the past tough 2 years.

**Comment by alperdemiral** [comment]
I agree 👍

**Comment by cryuwan** [comment]
As a community validator from the beginning of Avalanche mainnet I think that foundation nodes should run with higher commision rates than community validators.  Foundation nodes are important for the security of the network since they are staking huge amounts of tokens and addressed more than community nodes. On the other hand foundation nodes are earning majority of staking rewards. The foundation can support nodes running with less than 10 k avax. This could be via incentivizing these community validators who support the decentralization of the network without great profits. Governance can be activated and the minimum of staking fees should be increased in order to cover vps expenses etc.

**Comment by gatesyp** [comment]
why is this posted as an ACP? which track is this supposed to fall under? 

please check out [these definitions](https://github.com/avalanche-foundation/ACPs?tab=readme-ov-file#acp-tracks) for guidance

**Comment by izzetemredemir** [comment]
I find the proposal of the Avalanche Foundation staking AVAX tokens for validators quite valuable as it supports the validators efficiently. However, I'm curious about a couple of things: How much AVAX does the Avalanche Foundation currently hold? Also, are the validators operated by the Foundation known, and is there a way to view this information?
